modules/ROOT/pages/database-administration/standard-databases/manage-databases.adoc

```diff
@@ -0,0 +1,727 @@
+
+| `seedConfig`
+| comma separated list of configuration values.
+|
+Defines additional configuration specified by comma separated `name=value` pairs that might be required by certain seed providers.
+
+| `seedCredentials`
+| credentials
+|
+Defines credentials that needs to be passed into certain seed providers.
+
+|===
+
+[NOTE]
+====
+The `existingData`, `existingDataSeedInstance`, `seedURI`, `seedConfig`, and `seedCredentials` options cannot be combined with the `OR REPLACE` part of this command.
+More details about seeding options can be found in xref::clustering/databases.adoc#cluster-seed[Seed a cluster].
+====
+
+[role=enterprise-edition not-on-aura]
+[[manage-databases-alter]]
+== Alter database access mode
+
+The access mode of a standard database can be modified using the command `ALTER DATABASE`.
+
+By default, a database has read-write access mode on creation.
+The database can be limited to read-only mode on creation using the configuration parameters `dbms.databases.default_to_read_only`, `dbms.databases.read_only`, and `dbms.database.writable`.
+For details, see the section on xref::database-administration/standard-databases/configuration-parameters.adoc#[Configuration parameters].
+A database that was created with read-write access mode can be changed to read-only.
+To change it to read-only, you can use the `ALTER DATABASE` command with the sub-clause `SET ACCESS READ ONLY`.
+Subsequently, the database access mode can be switched back to read-write using the sub-clause `SET ACCESS READ WRITE`.
+Altering the database access mode is allowed at all times, whether a database is online or offline.
+
+If conflicting modes are set by the `ALTER DATABASE` command and the configuration parameters, i.e. one says read-write and the other read-only, the database will be read-only and prevent write queries.
+
+[NOTE]
+====
+Modifying access mode is only available to standard databases and not composite databases.
+====
+
+
+=== Alter database access mode to read-only
+
+.Query
+[source, cypher]
+----
+ALTER DATABASE customers SET ACCESS READ ONLY
+----
+
+The database access mode can be seen in the `access` output column of the command `SHOW DATABASES`.
+
+.Query
+[source, cypher]
+----
+SHOW DATABASES yield name, access
+----
+
+.Result
+[role="queryresult"]
+----
++----------------------------+
+| name        | access       |
++----------------------------+
+| "customers" | "read-only"  |
+| "movies"    | "read-write" |
+| "neo4j"     | "read-write" |
+| "system"    | "read-write" |
++----------------------------+
+----
+
+=== Alter database access mode to read-write
+
+`ALTER DATABASE` commands are optionally idempotent, with the default behavior to fail with an error if the database does not exist.
+Appending `IF EXISTS` to the command ensures that no error is returned and nothing happens should the database not exist.
+
+.Query
+[source, cypher]
+----
+ALTER DATABASE nonExisting IF EXISTS
+SET ACCESS READ WRITE
+----
+
```

Comment:
I assume topology was intentionally left out since it's missing here as well (and I vaguely remember it mentioned in the pr description?), let's hope no parts gets left behind when it gets moved over